src/bot.ts

```diff
@@ -18,6 +18,82 @@ import { is, not } from './utils'
 export const opencollective = (app: probot.Application): void => {
   app.log('OpenCollective Bot up!')
 
+  app.on('installation_repositories', async (context: probot.Context) => {
+    //get 'added' action as condition before adding the config file
+    if (context.payload.action === 'added') {
+      let added_repos = context.payload.repositories_added
+
+      // This can open PRs with new files at any point in time, but this
+      // gets called when the app is first installed on a repo
+      context.log({ event: context.event, action: context.payload.action })
+
+      let fields = {
+        file: {
```

Comment:
Maybe we should extract the yaml content in a separated file that we just read from the filesystem here.

Some ideas:
- most if not all options should contains the default values, maybe commented by default
- the file should be heavily commented to explain the different options to the user (I'll try to provide a good example of that, I have PHP or elasticsearch configuration?).

Reply:
Yeah, extracting to a separate file would be much better, I'll do that.

I'll try to add the default values, and an example would be very helpful, thanks.

Reply:
I was mentioning such configuration files. I don't think they are the best, but they're making a good job at explaining the features, and in the case of elasticsearch.yml the defaults.

- https://github.com/elastic/elasticsearch/blob/master/distribution/src/config/elasticsearch.yml
- https://github.com/php/php-src/blob/master/php.ini-production